web-client/integration-tests/journey/getUserMessageCount.js

```diff
@@ -0,0 +1,8 @@
+export const getUserMessageCount = async (cerebralTest, box, queue) => {
```

Comment:
I really like pulling this into a named file! Should this be moved into a "helpers" folder so we can add more functions like this in the same place in the future?

Reply:
I agree.. its been moved to the helper file